app/uk/gov/hmrc/cgtpropertydisposalsfrontend/models/BusinessPartnerRecord.scala

```diff
@@ -19,11 +19,11 @@ package uk.gov.hmrc.cgtpropertydisposalsfrontend.models
 import play.api.libs.json.{Format, Json}
 
 final case class BusinessPartnerRecord(
-    forename: String,
-    surname: String,
-    emailAddress: Option[String],
-    address: Address,
-    sapNumber: String
+  forename: String,
```

Comment:
you might need to get rid of `forename` and `surname` ub this case class? The BE won't be sending it in the response since you took it out

Reply:
Yes. Will modify the `SubscriptionDetails.fromBusinessPartnerRecord` accordingly to make the auth record the source of truth for the name of the individual

Reply:
refactored